app/src/main/java/com/daotrung/listcountry/model/Country.kt

```diff
@@ -0,0 +1,9 @@
+package com.daotrung.listcountry.model
+
+data class Country(
+    val id: Int,
+    val nameCountry: String,
```

Comment:
chỉ cần đặt là name là đủ 
```suggestion
    val name: String,
```